.circleci/config.yml

```diff
@@ -239,45 +239,76 @@ jobs:
+        - run: mv ./workbase/build/grakn-workbase-${GRAKN_VERSION}-linux.tar.gz ~/grakn/artifacts
         - persist_to_workspace:
             root: ~/grakn
             paths:
-              - ./workbase/build/
+              - artifacts/*-linux.tar.gz
 
   build-workbase-mac:
         macos:
             xcode: "9.0"
         working_directory: ~/grakn
         steps:
           - *attach_grakn_workspace
-          - *start_grakn_server
-          - run: cd grakn-dist/target/grakn-core-test/ && ./graql console -f ./examples/basic-genealogy.gql -k gene
+          - run: echo 'export GRAKN_VERSION=$(echo ${CIRCLE_TAG} | cut -c 2-)' >> $BASH_ENV #remove 'v' from tag name and use it as Grakn Version to be released
+          - run: unzip artifacts/grakn-core-${GRAKN_VERSION}.zip
+          - run: cd grakn-core-${GRAKN_VERSION} && nohup ./grakn server start
+          - run: cd grakn-core-${GRAKN_VERSION} && ./graql console -f ./examples/basic-genealogy.gql -k gene
           - run: cd workbase && npm install
           - run: cd workbase && npm run e2e
           - run: cd workbase && npm run build
+          - run: mv ./workbase/build/grakn-workbase-${GRAKN_VERSION}-mac.zip ~/grakn/artifacts
           - persist_to_workspace:
               root: ~/grakn
               paths:
-                - ./workbase/build/
+                - artifacts/*-mac.zip
 
-  publish-github-draft: #TO BE COMPLETED
+  publish-github-draft:
     docker:
       - image: cibuilds/github:0.10
     working_directory: ~/grakn
     steps:
       - attach_workspace:
           at: ~/grakn
       - run:
-          name: "Publish Release on GitHub"
+          name: "Publish Draft Release on GitHub"
           command: |
-            ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} ./grakn-dist/target/ ./workbase/build
+            ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete -draft ${CIRCLE_TAG} ./artifacts
+
+  grakn-release-build:
+    machine: true
+    working_directory: ~/grakn
+    steps:
+      - checkout
+      - *prepare_apt_repositories
+      - *install_yarn_and_maven
+      - *install_node
+      - *restore_m2_cache
+      - run: echo 'export GRAKN_VERSION=$(echo ${CIRCLE_TAG} | cut -c 2-)' >> $BASH_ENV #remove 'v' from tag name and use it as Grakn Version to be released
+      - run: mvn --batch-mode install -T 2.5C -DskipTests=true
+      - run: mkdir ~/grakn/artifacts
+      - run: mv grakn-dist/target/grakn-core-${GRAKN_VERSION}.zip ~/grakn/artifacts
+      - *save_m2_cache
+      - persist_to_workspace: #share Grakn with other jobs by putting it in the workspace
+          root: ~/grakn
+          paths:
+            - ./artifacts
+            - ./workbase
 
 workflows:
   version: 2
   run-all-tests:
     jobs:
-      - build
-      - workbase-unit-tests
+      - build:
+          filters:
+            tags:
+              ignore: /.*/
```

Comment:
question: what are we ignoring here?

Reply:
We ignore the workflow when we attach a tag to the current commit. 
We currently run all the tests every time someone submits a PR, so when we push the a commit with only a tag attached to it (and no changes) we only run the deployment workflow and we don't re-run all the tests again